src/VisualStudio/Core/Def/RoslynPackage.cs

```diff
@@ -152,16 +151,17 @@ protected override async Task InitializeAsync(CancellationToken cancellationToke
             // Ensure the options persisters are loaded since we have to fetch options from the shell
             LoadOptionPersistersAsync(this.ComponentModel, cancellationToken).Forget();
 
-            _workspace = this.ComponentModel.GetService<VisualStudioWorkspace>();
```

Comment:
no more creating hte workspace (which creates its entire mef composition on this thread).